SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -88,6 +88,26 @@ void UpdateChangelistHistory(TUniquePtr<FRepState>& RepState)
 	SendingRepState->HistoryStart = SendingRepState->HistoryStart % MaxSendingChangeHistory;
 	SendingRepState->HistoryEnd = SendingRepState->HistoryStart + NewHistoryCount;
 }
+
+void ForceReplicateOnActorHierarchy(USpatialNetDriver* NetDriver, AActor* RootActor, const AActor* OriginalActor)
+{
+	if (RootActor->GetIsReplicated())
+	{
+		if (RootActor != OriginalActor)
+		{
+			if (USpatialActorChannel* Channel = NetDriver->GetActorChannelByEntityId(NetDriver->PackageMap->GetEntityIdFromObject(RootActor)))
```

Comment:
Should we just call GetOrCreateSpatialActorChannel here instead?

Reply:
What would it mean to reach that point without having an ActorChannel ? Is it expected to be lazily created ?

Reply:
Impossible for us to be at the stage of considering authority intent migration without having an Actor channel so there's no need, idm changing if it's a stylistic preference

Reply:
Wouldn't it just require a new replicated actor that hasn't been replicated yet, before it's inheritance chain attempts to migrate?

Reply:
ah yea, this before I used my brain and realised this function is applied to all hierarchy Actors

Reply:
Cool - so I think we should swap this with `GetOrCreateSpatialActorChannel`